flower/utils/pg_storage.py

```diff
@@ -18,16 +18,40 @@
     (
         id SERIAL PRIMARY KEY,
         time TIMESTAMP NOT NULL,
+        uuid VARCHAR(36) NOT NULL,
         data JSONB NOT NULL,
```

Comment:
could store as TEXT instead of JSONB, since we're not using a GIN index... maybe get some more perf by not validating the json?

Reply:
Dropping the GIN index and the JSONB is not an option for me, as I already depend on those for some data mining that is separate from Flower -- and I expect there will more of that in the future.